dts/xtensa/espressif/esp32s3.dtsi

```diff
@@ -202,6 +203,24 @@
 			status = "disabled";
 		};
 
+		adc0: adc@3f440018 {
```

Comment:
Does anybody know that the base peripheral address is for the ADCs?  I copied `3f440018` from the ESP32-S2.  Since it isn't used in the adc_esp32.c driver, it doesn't matter, but it should still be correct.

Reply:
according to the -S3 TRM it is `0x60040000`

Reply:
Awesome, thank you.  System & Memory section in the TRM, will remember that in the future now ;)